nemo/collections/tts/modules/radtts.py

```diff
@@ -464,10 +463,11 @@ def forward(
 
         f0_bias = 0
         # unvoiced bias forward pass
+        voiced_mask_bool = voiced_mask.bool()
```

Comment:
Line 385 and 400: functions `binarize_attention` and `get_first_order_features` should add decor `@staticmethod`, or perhaps there are better places to host such generic functions. 

Besides, `out_lens` in `get_first_order_features` is not used anywhere, please remove.

please also update docstring for the two functions.

Reply:
line 439: `p_voiced=None` is not used in the `forward()`. pls remove.

Reply:
Those cleanups are not in any way related to the changes I was doing - I would rather avoid any changes that are not necessary.

Reply:
ok. We can have a separate PR for clean up.